### ISSUE TITLE
Replaying cmdlog on stdin doesn't work

### ISSUE THREAD

**Issue by niko** [saulpw/visidata#124]
Given this cmdlog:
```
sheet	col	row	keystrokes	input	comment
			o	stdin	open file
stdin	2		#		set type of current column to int
```
visidata throws this error on replay:
`FileNotFoundError": [Erno 2] No such file or directory: stdin`

**Comment by saulpw** [comment]
Hi @niko, try using `-` instead of `stdin` (this was just implemented in v1.1).

**Comment by saulpw** [comment]
Fixed as of 76036db50b87d9ace72cecf85a86f68a41c67bea

**Comment by saulpw** [comment]
Thanks for reporting this issue, @niko.  It was more complicated than I thought, but it should all work now.  This will be included in v1.2.  Let me know if you have any other issues.
